package.json

```diff
@@ -51,12 +51,17 @@
     "storybook-readme": "^4.0.0-beta1"
   },
   "jest": {
-    "resetModules": true,
-    "testMatch": [
-      "**/?(*.)(spec).js?(x)"
+    "collectCoverageFrom": [
+      "**/(src|scripts)/**/*.{js,jsx}",
+      "!**/src/app/helpers/tests/**",
```

Comment:
Does this make strange assumptions about the structure of the file system? I think especially L56, the others seem fairly agnostic. 🤔

Reply:
Oh, it does! Well spotted. Can probably delete this line.